.github/workflows/ci.yml

```diff
@@ -73,7 +76,33 @@ jobs:
       - name: Run Tests
         working-directory: target/snitch_cluster
         run: |-
-          ./run.py sw/snax-mac-run.yaml --simulator verilator
+          ./run.py --simulator verilator \
+          sw/runtime.yaml sw/custom-fp.yaml sw/standard-fp.yaml \
+          sw/openmp.yaml sw/blas.yaml sw/dnn.yaml sw/snax-mac-run.yaml -j
+
+  sw-snax-mac-cluster-vlt-generic:
+    name: Simulate SW on SNAX MAC Cluster w/ Verilator (Generic LLVM)
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/kuleuven-micas/snax:latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Build Hardware
+        run: |
+          make CFG_OVERRIDE=cfg/snax-mac.hjson \
+          -C target/snitch_cluster bin/snitch_cluster.vlt
+      - name: Build Software
+        run: |
+          make -C target/snitch_cluster sw \
+          SELECT_RUNTIME=rtl-generic \
+          SELECT_TOOLCHAIN=llvm-generic
+      - name: Run Tests
+        working-directory: target/snitch_cluster
+        run: |-
+          ./run.py --simulator verilator \
+          sw/runtime.yaml sw/standard-fp.yaml sw/snax-mac-run.yaml -j
```

Comment:
It dawned on me but I wonder if we can re-use the same `sw-snax-mac-cluster-vlt` and just build a new SW so we don't need to do multiple HW builds?

So the idea is, build SW for the original one then build for the generic one. Test them differently since we already have the yamls separated? :)

Reply:
I created #67